test/BandPriceFeed.spec.ts

```diff
@@ -88,14 +88,15 @@ describe("BandPriceFeed Spec", () => {
             expect(observation.priceCumulative).to.eq(parseEther("6000"))
         })
 
-        it("force error, the second update is the same timestamp", async () => {
+        // skip this test for being compatible with Chainlink aggregator
```

Comment:
But Chainlink also uses `CumulativeTwap._update()`, right? So... I guess we cannot skip it

Reply:
skip this because the timestamp from chainlink aggregator in different round might be the same. check here,  
https://github.com/perpetual-protocol/perp-oracle/pull/11/files#diff-bae889170f4f6b77f19bc28ee4e7e1cb143606a6a3e4e76531aa689769d1d36cR40

Does it make sense to you?

Reply:
Right, thx for the clarafication!